airbyte-integrations/connectors/source-fleetio/build.gradle

```diff
@@ -0,0 +1,9 @@
+plugins {
```

Comment:
Yeah, let's delete Gradle build from here. We're walking away from using Gradle for anything Python.